.pre-commit-config.yaml

```diff
@@ -17,88 +17,13 @@ exclude: |
 default_language_version:
   python: python3
 repos:
-  - repo: local
-    hooks:
-      # These files are most likely copier diff rejection junks; if found,
-      # review them manually, fix the problem (if needed) and remove them
-      - id: forbidden-files
-        name: forbidden files
-        entry: found forbidden files; remove them
-        language: fail
-        files: "\\.rej$"
   - repo: https://github.com/oca/maintainer-tools
     rev: ab1d7f6
     hooks:
       # update the NOT INSTALLABLE ADDONS section above
       - id: oca-update-pre-commit-excluded-addons
       - id: oca-fix-manifest-website
         args: ["https://github.com/OCA/cooperative"]
-  - repo: https://github.com/myint/autoflake
-    rev: v1.4
-    hooks:
-      - id: autoflake
-        args:
-          - --expand-star-imports
-          - --ignore-init-module-imports
-          - --in-place
-          - --remove-all-unused-imports
-          - --remove-duplicate-keys
-          - --remove-unused-variables
-  - repo: https://github.com/psf/black
-    rev: 20.8b1
-    hooks:
-      - id: black
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.1.2
-    hooks:
-      - id: prettier
-        name: prettier (with plugin-xml)
-        entry: prettier --write --list-different
-        additional_dependencies:
-          - "prettier@2.1.2"
-          - "@prettier/plugin-xml@0.12.0"
-        files: \.(css|htm|html|js|json|jsx|less|md|scss|toml|ts|xml|yaml|yml)$
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v6.8.0
-    hooks:
-      - id: eslint
-        verbose: true
-        args:
-          - --color
-          - --fix
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
-    hooks:
-      - id: trailing-whitespace
-        # exclude autogenerated files
-        exclude: /README\.rst$|\.pot?$
-      - id: end-of-file-fixer
-        # exclude autogenerated files
-        exclude: /README\.rst$|\.pot?$
-      - id: debug-statements
-      - id: fix-encoding-pragma
-        args: ["--remove"]
-      - id: check-case-conflict
-      - id: check-docstring-first
-      - id: check-executables-have-shebangs
-      - id: check-merge-conflict
-        # exclude files where underlines are not distinguishable from merge conflicts
-        exclude: /README\.rst$|^docs/.*\.rst$
-      - id: check-symlinks
-      - id: check-xml
-      - id: mixed-line-ending
-        args: ["--fix=lf"]
```

Comment:
I kind of don't like the removal of these. We _could_ manually keep these, but I'm not sure if that's the proper thing to do.

Reply:
Can I merge anyway or do you want to fix something ?